src/Application.cc

```diff
@@ -598,8 +607,14 @@ std::vector<std::pair<std::string, std::vector<std::string>>>
       // All we verify is that the file starts with "lib", any further
       // checks would require loading the plugin.
 
+#ifndef _WIN32
```

Comment:
nit: I would find it easier to read with reversed logic:

~~~
#ifdef _WIN32
      if (plugin.find(".lib") != std::string::npos)
#else
      if (plugin.find("lib") == 0)
#endif
~~~

Reply:
reversed in 3b0a96b